eZ/Bundle/EzPublishCoreBundle/Imagine/VariationPathGenerator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
```

Comment:
ping @bdunogier was this header fix missed? This seems important and part of the CS that we all have come to expect from / in eZ code ...